dozer-api/src/generator/protoc/generator/implementation.rs

```diff
@@ -167,15 +167,32 @@ impl<'a> ProtoGeneratorImpl<'a> {
                 })
         }
 
-        fn record_desc_from_message(
-            message: MessageDescriptor,
-        ) -> Result<RecordDesc, GenerationError> {
-            let version_field = get_field(&message, "__dozer_record_version")?;
-            Ok(RecordDesc {
-                message,
-                version_field,
-            })
-        }
+        let record_desc_from_message =
+            |message: MessageDescriptor| -> Result<RecordDesc, GenerationError> {
+                let version_field = get_field(&message, "__dozer_record_version")?;
+                let point_values = descriptor
+                    .get_message_by_name("dozer.types.PointType")
```

Comment:
`dozer.types.PointType` and `dozer.types.RustDecimal` are hardcoded in two different places. Better create constants for them.

It's better if `unwrap` can be replaced by an error with proper context.

Reply:
alright